src/components/pageGrid.tsx

```diff
@@ -53,11 +66,21 @@ export default ({ nextPages = false, header }: Props): JSX.Element => {
 
   if (!matches.length) return null;
 
+  const addedPages =
+    (additionalPages &&
```

Comment:
if we default `additionalPages` to be an empty array we don't need this check.

Reply:
Good idea, added in [6f1dc8d](https://github.com/getsentry/sentry-docs/pull/5625/commits/6f1dc8d6460e713d502f683b097bf7e22b4faa55)